WEATHERAPP/Components/Login/LoginPage.css

```diff
@@ -0,0 +1,112 @@
+body
+{
+   margin: 0;
+}
+.background
+{
+   height: 100vh;
+   background-image:url("../../Assets/L&D-BG\ 4.png");
+   display: flex;
+   flex-direction: column;
+   justify-content: space-evenly;
+}
+.app-header
+{
+   text-align: center;
+   font:700 72px system-ui;
+}
+.main-contents
+{
+   display: inline-flex;
+   justify-content: space-around;
+}
+.weather-image
+{
+   width: 300px;
+   height: 300px;
+}
+.login-form
+{
+   display: flex;
+   flex-direction: column;
+   align-items: center;
+   justify-content: space-evenly;
+   background-color: rgba(132, 95, 95, 1);
```

Comment:
Add the colors in a separate file, called colors .css and try to use here

Reply:
resolved